packages/longhorn/longhorn.patch

```diff
@@ -49,6 +49,7 @@ diff -x '*.tgz' -x '*.lock' -uNr packages/longhorn/charts-original/Chart.yaml pa
 +  catalog.cattle.io/ui-component: longhorn
 +  catalog.cattle.io/provides-gvr: longhorn.io/v1beta1
```

Comment:
is this being used by another chart?

Reply:
Not that I'm aware of, we do it preemptively though.